test/unit/test-amp-story-player.js

```diff
@@ -691,58 +689,93 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
       return expect(() => player.go(-1)).to.throw('Out of Story range.');
     });
 
-    it('signals when player changed story using next method', async () => {
+    it('takes to first story when swiping on the last one with circular wrapping', async () => {
       const playerEl = win.document.createElement('amp-story-player');
       appendStoriesToPlayer(playerEl, 5);
 
       const player = new AmpStoryPlayer(win, playerEl);
 
       await player.load();
+      await nextTick();
 
       const navigationSpy = env.sandbox.spy();
       playerEl.addEventListener('navigation', navigationSpy);
-      player.next_();
-      expect(navigationSpy.firstCall.args[0].type).to.eql('navigation');
-      expect(navigationSpy.firstCall.args[0].detail).to.eql({
-        index: 1,
-        remaining: 3,
+      playerEl.setAttribute('enable-circular-wrapping', '');
```

Comment:
I think it's better to add this attribute right after we create the element, so we simulate a bit closer the real world scenario